zeppelin-web/src/components/noteName-create/notename.controller.js

```diff
@@ -14,9 +14,10 @@
 
 'use strict';
 
-angular.module('zeppelinWebApp').controller('NotenameCtrl', function($scope, $rootScope, $routeParams, websocketMsgSrv,
-                                                                     $location) {
+angular.module('zeppelinWebApp').controller('NotenameCtrl', function($scope, notebookListDataFactory,
+                                                             $rootScope, $routeParams, websocketMsgSrv) {
   var vm = this;
+  vm.notes = notebookListDataFactory;
   vm.websocketMsgSrv = websocketMsgSrv;
   $scope.note = {};
```

Comment:
All of the controller variables should be defined at the top, so we should have
`vm.clone = false;` here too